hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieCatalog.java

```diff
@@ -352,12 +352,13 @@ public void createTable(ObjectPath tablePath, CatalogBaseTable catalogTable, boo
     }
     conf.setString(FlinkOptions.TABLE_NAME, tablePath.getObjectName());
     try {
-      StreamerUtil.initTableIfNotExists(conf);
+      HoodieTableMetaClient metaClient = StreamerUtil.initTableIfNotExists(conf);
       // prepare the non-table-options properties
       if (!StringUtils.isNullOrEmpty(resolvedTable.getComment())) {
         options.put(TableOptionProperties.COMMENT, resolvedTable.getComment());
       }
       TableOptionProperties.createProperties(tablePathStr, hadoopConf, options);
+      HoodieCatalogUtil.initPartitionBucketIndexMeta(metaClient, catalogTable);
```

Comment:
Not sure how the consistent hashing meta is initialized, can we follow that?

Reply:
consistent hashing meta create and save meta during write process.
For partition bucket index,  Hashing_Config can only be created in DDL and Call commands, but not at runtime